### PR TITLE
[ZEPPELIN-2700] Provide a link to interpreter binding mode doc

### PR DESCRIPTION
**Pull request by 1ambda**
### What is this PR for?

Provide a link to the interpreter binding mode documentation page in the interpreter page to help understand what this feature is for (`usage/interpreter/interpreter_binding_mode.html`)

- **documentation is not deployed to web yet. So clicking this icon will redirect you to 404 page**

### What type of PR is it?
[Improvement | Documentation]

### Todos

DONE

### What is the Jira issue?

[ZEPPELIN-2700](https://issues.apache.org/jira/browse/ZEPPELIN-2700)

### How should this be tested?

1. Open the interpreter page 
2. Check the links (URL, icon, ..)

#### Before

image

image

#### After

image

image

### Questions:
* Does the licenses files need update? - NO
* Is there breaking changes for older versions? - NO
* Does this needs documentation? - NO

### CODE REVIEW

**Comment by 1ambda** [comment]
CI failed but irrelevant.

- https://travis-ci.org/1ambda/zeppelin/jobs/247790231

```java
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 145.149 sec - in org.apache.zeppelin.integration.AuthenticationIT

Results :

Tests in error: 
  ZeppelinIT.testSparkInterpreterDependencyLoading:249->AbstractZeppelinIT.waitForParagraph:68->AbstractZeppelinIT.pollingWait:94 Â» Timeout

Tests run: 24, Failures: 0, Errors: 1, Skipped: 0
```

**Comment by 1ambda** [comment]
Ready to review.

**Comment by 1ambda** [comment]
Updating doc is discussed in https://github.com/apache/zeppelin/pull/2437

File: zeppelin-web/src/app/interpreter/interpreter-create/interpreter-create.html [hunk 1]
```diff
@@ -125,7 +124,11 @@
                 </li>
               </ul>
             </span>
-              process.
+              process
+              <a class="fa fa-info-circle interpreter-binding-mode-info-link"
+                 aria-hidden="true"
+                 uib-tooltip="click"
```
Comment (felixcheung) [inline zeppelin-web/src/app/interpreter/interpreter-create/interpreter-create.html]: probably something more descriptive for tooltip?

File: zeppelin-web/src/app/interpreter/interpreter.html [hunk 2]
```diff
@@ -302,7 +303,11 @@ <h3 class="interpreter-title">{{setting.name}}
                   </li>
                 </ul>
               </span>
-              process.
+              process
+              <a class="fa fa-info-circle interpreter-binding-mode-info-link"
+                 aria-hidden="true"
+                 uib-tooltip="click"
```
Comment (felixcheung) [inline zeppelin-web/src/app/interpreter/interpreter.html]: ditto here

File: zeppelin-web/src/app/interpreter/interpreter.html [hunk 3]
```diff
@@ -240,7 +240,11 @@ <h3 class="interpreter-title">{{setting.name}}
                 </li>
               </ul>
             </span>
-            process.
+            process
+            <a class="fa fa-info-circle interpreter-binding-mode-info-link"
+               aria-hidden="true"
+               uib-tooltip="click"
```
Comment (felixcheung) [inline zeppelin-web/src/app/interpreter/interpreter.html]: ditto

File: zeppelin-web/src/app/interpreter/interpreter-create/interpreter-create.html (hunk 1 above)
Comment (1ambda) [inline zeppelin-web/src/app/interpreter/interpreter-create/interpreter-create.html]: It would be better to do that.

**Comment by 1ambda** [comment]
Thanks for the review @felixcheung, I updated the tooltip message.

image

**Comment by 1ambda** [comment]
merge if no more discussion

**Pull request closed without merging**